membership/index.html

```diff
@@ -1,62 +1,76 @@
   <li><strong>3D prototyping facilities</strong></li>
+  <li><strong>Kitted-out workshop</strong> with a growing selection of hand tools &amp; machines</li>
   <li><strong>Free entry</strong> to all Farset Labs events<sup id="fnref:events"><a href="#fn:events" class="footnote">1</a></sup></li>
   <li><strong>Free workshops</strong> on all sorts of topics, from soldering to accounting</li>
-  <li><strong>Networking opportunities</strong> with the best and brightest in Belfast</li>
-  <li><strong>Belonging</strong> to an intelligent and creative community</li>
 </ul>
 
-<hr>
+<img src="/membership/membership_800.jpg" alt="Membership">
+
+<br>
+
 <h1 id="membership-dues">Membership Types</h1>
 
 <p>There are three Individual Membership rates that we think represent three core uses of the space; </p>
 
 <ul>
   <li><strong>Students</strong> (£15pm), that are occasionally in the space during the day, mostly experiment in the evenings, and are strapped for cash <sup id="fnref:cards"><a href="#fn:cards" class="footnote">2</a></sup></li>
-  <li><strong>Enthusiasts</strong> (£25pm), that regularly use the space in the evenings and make regular use of the equipment and space, less strapped for cash</li>
+  <li><strong>Enthusiasts</strong> (£25pm), that regularly use the space in the evenings and make regular use of the equipment and space</li>
   <li><strong>Freelancers</strong> (£35pm), that are consistently in the space, regularly hot-desking with ongoing long term projects under their belts</li>
 </ul>
 
-<form class="center" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+<form class="center small-8" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
+<input type="hidden" name="on0" value="">
 <input type="hidden" name="cmd" value="_s-xclick">
 <input type="hidden" name="hosted_button_id" value="XJ9H6BLXWWYXL">
 <table>
-<tr><td><input type="hidden" name="on0" value=""></td></tr><tr><td><select name="os0">
-  <option value="Student (Monthly)">Student (Monthly) : £15.00 GBP - monthly</option>
-  <option value="Enthusiast (Monthly)">Enthusiast (Monthly) : £25.00 GBP - monthly</option>
-  <option value="Freelancer (Monthly)">Freelancer (Monthly) : £35.00 GBP - monthly</option>
-  <option value="Angel (Monthly)">Angel (Monthly) : £75.00 GBP - monthly</option>
-  <option value="Student (Annual)">Student (Annual) : £165.00 GBP - yearly</option>
-  <option value="Enthusiast (Annual)">Enthusiast (Annual) : £275.00 GBP - yearly</option>
-  <option value="Freelancer (Annual)">Freelancer (Annual) : £385.00 GBP - yearly</option>
-</select> </td></tr>
+<tr>
+  <td>
+    <select name="os0">
+      <option value="Student (Monthly)">Student (Monthly) : £15.00 GBP</option>
+      <option value="Enthusiast (Monthly)">Enthusiast (Monthly) : £25.00 GBP</option>
+      <option value="Freelancer (Monthly)">Freelancer (Monthly) : £35.00 GBP</option>
```

Comment:
I thought we were just doing "Student" and "regular" membership now, rather than Enthusiast/Freelancer levels?